### DISCUSSION TITLE
`match_score` and `matching_score` in `fuzzy_join` are confusing

### DISCUSSION THREAD

**Discussion by jeromedockes** [skrub-data/skrub#803] [General]
in `fuzzy_join`, `matching_score` refers to the score obtained by each row (eg a `matching_score` column is added when `return_score=True`) and `match_score` is the minimum matching score for a match to be accepted. The 2 names are very similar and it is not clear how the terms' meaning differ making it hard to remember which is which. maybe better names would be `match_score` and `min_match_score`, `match_score_threshold` or something similar

**Comment by GaelVaroquaux** [comment]
Agreed with the confusing, but the replacements proposal are too long.

**Comment by jeromedockes** [comment]
maybe `score` and `min_score` then?

**Reply by GaelVaroquaux** [reply]
Yes, that great!

**Comment by jeromedockes** [comment]
has been fixed in #821
